internal/validations/clusterlogforwarder/outputs/splunk_test.go

```diff
@@ -43,6 +43,15 @@ var _ = Describe("Validate Splunk:", func() {
 			Expect(valid).To(BeFalse())
 			Expect(map[string]status.Condition{"splunk": st}).To(HaveCondition("Ready", false, loggingv1.ReasonInvalid, "output \"splunk\": IndexKey can only contain letters, numbers, and underscores \\(a-zA-Z0-9_\\). Segments that contain characters outside of this range must be quoted."))
 		})
+
+		It("should fail if Fields is spec'd", func() {
```

Comment:
Can we pass validations but provide the warning.  My concern here is the upgrade scenario.  What will happen if this is defined and we fail it

Reply:
How do you feel about passing the validation, but setting condition of Degraded=True?